src/components/TokenOptionItem.tsx

```diff
@@ -153,19 +153,16 @@ const generateMessage = ({
   switch (reason) {
     // not registered --> advise to register
     case TokenFromExchange.NOT_REGISTERED_ON_CONTRACT:
-      if (!token)
+      if (!token) {
```

Comment:
😤 :stuck_out_tongue_closed_eyes:

Reply:
what?? :) 
Do you really think is better to skip the brackets?